server/models/Column.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const columnSchema = new mongoose.Schema(
+  {
+    name: {
+      type: Array,
+      required: false,
+    },
+    cards: {
+      type: Array,
+      required: true,
+    },
+  },
+  { timestamps: true }
+);
+
+columnSchema.methods.removePassword = function () {
+  const object = this.toObject();
+  delete object.user.password;
+  return object;
+};
```

Comment:
Same as the previous comment. User doesn't get referenced, thus don't need to worry about the user's password